.travis.yml

```diff
@@ -7,49 +7,55 @@ cache:
     - tmp/cache/assets/test/sprockets
 env:
   matrix:
-    - GROWSTUFF_ELASTICSEARCH='true' RSPEC_TAG=elasticsearch
-    - GROWSTUFF_ELASTICSEARCH='false' RSPEC_TAG=~elasticsearch
-    - STATIC_CHECKS='true'
+    - GROWSTUFF_ELASTICSEARCH='true' RSPEC_TAG=elasticsearch STATIC_CHECKS=false
+    - GROWSTUFF_ELASTICSEARCH='false' RSPEC_TAG=~elasticsearch STATIC_CHECKS=false
+    - STATIC_CHECKS=true
   global:
     - GROWSTUFF_SITE_NAME="Growstuff (travis)"
     - RAILS_SECRET_TOKEN='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
     - secure: "Z5TpM2jEX4UCvNePnk/LwltQX48U2u9BRc+Iypr1x9QW2o228QJhPIOH39a8RMUrepGnkQIq9q3ZRUn98RfrJz1yThtlNFL3NmzdQ57gKgjGwfpa0e4Dwj/ZJqV2D84tDGjvdVYLP7zzaYZxQcwk/cgNpzKf/jq97HLNP7CYuf4="
 rvm:
   - 2.3.3
 before_install:
-  - export PATH=$PWD/travis_phantomjs/phantomjs-2.1.1-linux-x86_64/bin:$PATH
   - >
-    if [ $(phantomjs --version) != '2.1.1' ]; then
-    PHANTOM_URL=https://assets.membergetmember.co/software/phantomjs-2.1.1-linux-x86_64.tar.bz2;
-    rm -rf $PWD/travis_phantomjs;
-    mkdir -p $PWD/travis_phantomjs;
-    wget $PHANTOM_URL -O $PWD/travis_phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2;
-    tar -xvf $PWD/travis_phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2 -C $PWD/travis_phantomjs;
-    fi
-  - phantomjs --version
+    ./script/install_phantomjs;
+    phantomjs --version
+  - export PATH=$PWD/travis_phantomjs/phantomjs-2.1.1-linux-x86_64/bin:$PATH
 before_script:
   - >
-    if [ "$STATIC_CHECKS" != 'true' ]; then
-      bundle exec rake db:create db:migrate db:test:prepare
-      bundle exec rake assets:precompile
-    fi
+    set -e;
+    if [ "${STATIC_CHECKS}" = "true" ]; then
+      gem install overcommit rubocop haml-lint;
+      overcommit --install;
+      overcommit --sign;
+      overcommit --sign pre-commit;
+      # pip install yamllint;
+      npm install -g eslint jshint coffeelint csslint;
```

Comment:
Our dependencies are now split across `Gemfile` and `.travis.yml`. Are there gems we can depend on wrapping these tools?